{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/config.py

```diff
@@ -127,6 +127,10 @@ def _(x):
 #: route correct hosts to the application.
 APP_ALLOWED_HOSTS = ['localhost']
 
+# OAI-PMH
+# =======
+OAISERVER_ID_PREFIX = 'oai:{{cookiecutter.project_shortname}}.com:'
```

Comment:
Hmm, would it make sense to ask the user for the website of the project instead of assuming that the URL will always end with`.com` (I see that we also assume the `.com` for `JSONSCHEMA_HOST` config variable)?